makefile

```diff
@@ -0,0 +1,18 @@
+all: bin/bot
```

Comment:
Wish: it would be great to have a `help` section 🙏

Reply:
Done